src/sass/partials/components/_content-card.scss

```diff
@@ -0,0 +1,30 @@
+.card {
+  border: setting-get('input border width') solid setting-get('secondary color');
+  display: flex;
+  flex-direction: column;
+  margin-top: (setting-get('rhythm') / .5);
+  padding: setting-get('rhythm') / .75;
+
+  &--link {
+    @include link(setting-get('active link color'), setting-get('active link color'), setting-get('secondary link color'), setting-get('secondary link color'));
```

Comment:
This is identical to `.base--link`. Instead of duplicating this, add a `base-link` mixin to https://github.com/punchcard-cms/punchcard/blob/master/src/sass/partials/global/_mixins.scss with these defaults (or, alternatively, update the `link` mixin with these as the default arguments) and make sure that `.base--link` calls the mixin